tests/functions/test_uploads.py

```diff
@@ -179,12 +182,12 @@ def test_ingest_upload(capsys, monkeypatch):
 
     # Check that the job status was updated to reflect a successful upload
     assert job.status == UploadJobStatus.MERGE_COMPLETED.value
-    assert email_was_sent(capsys.readouterr()[0])
+    assert email_was_sent(caplog.records)
```

Comment:
It'd be a little more straightforward to pass `caplog.text` here and leave `email_was_sent` unchanged. `caplog.text` includes all logged text in one string.

Reply:
in next commit